src/gwt/src/org/rstudio/core/client/dom/DomUtils.java

```diff
@@ -1152,6 +1152,21 @@ public static void setPlaceholder(TextBox w, String placeholder)
       setPlaceholder(w.getElement(), placeholder);
    }
 
+   /**
+    * Set disabled attribute on an element's child
+    * @param element
+    * @param ordinal
+    * @param disable
```

Comment:
Doxygen style params should be either documented (i.e. explain what each one is) or omitted.

Reply:
Updated